lib/xen_os.mli

```diff
@@ -47,10 +47,23 @@ module Memory : sig
   (** Memory allocation statistics. Units are the system word size, as used by
       the OCaml stdlib Gc module. *)
 
+  val stat: unit -> stat
+  (** [stat ()]  returns memory allocation statistics. This call is slower
+      than quick_stat, and the returned values are accurate. *)
+
   val quick_stat : unit -> stat
   (** [quick_stat ()] returns memory allocation statistics. This call is
       computationally cheap, but the returned values may not be completely
       accurate. *)
+
+  val memory_usage_stat: unit -> stat
+  (** [memory_usage_stat ()]  returns memory allocation statistics. This call is
+      computationally cheap, and the returned values (even not be completely
+      accurate) should be more precise thant quick_stat. *)
```

Comment:
This uses `malloc_memory_usage`, which is precomputed (as `quick_stat`), but more precise.